CMakeLists.txt

```diff
@@ -93,7 +93,7 @@ set_property(CACHE RMM_LOGGING_LEVEL PROPERTY STRINGS "TRACE" "DEBUG" "INFO" "WA
                                               "CRITICAL" "OFF")
 message(STATUS "RMM: RMM_LOGGING_LEVEL = '${RMM_LOGGING_LEVEL}'")
 
-if(BUILD_TESTS OR BUILD_BENCHMARKS)
+if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME AND (BUILD_TESTS OR BUILD_BENCHMARKS))
```

Comment:
Can we refactor the conditions to be `if ( <primary_check> AND CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME )`. I like having the primary condition ( is testing on ) to be the first check as it conveys to readers what we are looking for primarily.

Reply:
Done.